src/main/java/cs2103/v15_1j/jimjim/command/ChangeCommand.java

```diff
@@ -67,6 +67,7 @@ public LocalTime getNewEndTime() {
 	public UIFeedback undo(ControllerStates conStates) {
 		TaskEvent temp = conStates.masterList.remove(actual);
 		conStates.masterList.add(taskNum-1, backup);
+        conStates.masterList.sort();
```

Comment:
Actually this can be done in delete/add commands as well. We don't really need to use the positions syntax which does not guarantee the list is sorted.